libsecret.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Any reason to store it in a separate file?

Reply:
Not really, just so it is more in line with the old `- shared-modules/libsecret/libsecret.json`